Dockerfile

```diff
@@ -1,12 +1,23 @@
-FROM centos:centos7
+# Default to EL7 builds
+ARG IMAGE_BASE_TAG=centos7
+
+FROM centos:$IMAGE_BASE_TAG
+
+# "ARG IMAGE_BASE_TAG" needs to be here again because the previous instance has gone out of scope.
+ARG IMAGE_BASE_TAG=centos7
 
 LABEL maintainer OSG Software <help@opensciencegrid.org>
 
-RUN yum update -y && \
+RUN [[ $IMAGE_BASE_TAG == centos7 ]] && \
+       YUM_PKG_NAME="yum-plugin-priorities" || \
+       YUM_PKG_NAME="yum-utils" && \
+    yum update -y && \
     yum -y install http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
                    epel-release \
-                   yum-plugin-priorities && \
+                   $YUM_PKG_NAME && \
     yum -y install supervisor cronie && \
+    [[ $IMAGE_BASE_TAG == centos7 ]] || yum-config-manager --enable PowerTools && \
+    [[ $IMAGE_BASE_TAG == centos7 ]] || yum-config-manager --enable osg-testing && \
```

Comment:
```suggestion
    if [[ $IMAGE_BASE_TAG != centos7 ]]; then \
        yum-config-manager --enable PowerTools && \
        yum-config-manager --enable osg-testing; \
    fi; \
```

Reply:
Shouldn't that be

```
fi && \
```

to retain the same logic?